modules/light-clients/08-wasm/types/client_state_test.go

```diff
@@ -173,17 +173,17 @@ func (suite *TypesTestSuite) TestValidate() {
 	}{
 		{
 			name:        "valid client",
-			clientState: types.NewClientState([]byte{0}, []byte{0}, clienttypes.ZeroHeight()),
+			clientState: types.NewClientState([]byte{0}, []byte("01234567012345670123456701234567"), clienttypes.ZeroHeight()),
```

Comment:
We can make this magic `01234567012345670123456701234567` string a constant somewhere. I am happy to do it in a follow-up PR to speed up merging this one.

Reply:
where does this magic string come from? 

Would be nice to move away from hardcoded values in testing in the long term, but I think we can also work on that post-release.

Reply:
This magic string just needs to be any 32 bytes to pass ClientState's Validate() function.

Reply:
Aha, makes sense, thanks @misko9